### ISSUE TITLE
ts-node -i repl do not fflush stdout

### ISSUE THREAD

**Issue by wsdjeg** [TypeStrong/ts-node#1013]
hello, and thanks for working for this project, and I just want to add typescript repl support in SpaceVim

the code runner works well now, but the repl do not work in my plugin, as the `ts-node -i` do not fflush the sdtout buffer until we send (os/exit).

**Comment by blakeembrey** [comment]
@wsdjeg Can you briefly describe what you're trying to do? The REPL is written to stdout here: https://github.com/TypeStrong/ts-node/blob/2a0f1f3b8f979dbc61726017f5de45a8b03f3c95/src/bin.ts#L407-L408 (it's initialized here: https://github.com/TypeStrong/ts-node/blob/2a0f1f3b8f979dbc61726017f5de45a8b03f3c95/src/bin.ts#L343-L345). There might be some options to tweak here.

**Comment by wsdjeg** [comment]
what is `terminal: process.stdout.isTTY,`? I do not use ts-node in terminal but in pipe.

**Comment by cspotcode** [comment]
@wsdjeg it's an option passed to `repl.start`, explained here: https://nodejs.org/docs/latest-v13.x/api/repl.html#repl_repl_start_options

**Comment by cspotcode** [comment]
Quick reproduction.  Simulate typing "foo<Enter>" into the REPL when stdout is not a TTY.  Error message is not emitted till ts-node terminates, after 10 seconds.

```
node -e 'console.log("foo");setTimeout(() => {}, 10e3)' | ts-node -i &> output.txt
```
In another shell:
```
watch -n 0.5 cat output.txt
```

**Comment by cspotcode** [comment]
@wsdjeg This should be fixed by #1019, which has been released in [v8.9.1](https://github.com/TypeStrong/ts-node/releases/tag/v8.9.1)

Thank you for the bug report!

**Comment by wsdjeg** [comment]
thanks
